__tests__/app.test.js

```diff
@@ -167,5 +167,13 @@ describe('/api/articles/:article_id/comments', () => {
             .then(({ body: { msg } }) => {
             expect(msg).toBe('Bad request')
         })
-    })  
+    })
+    test('GET 200: responds with an empty array if the article_id exists but has no comments', () => {
+        return request(app)
+        .get('/api/articles/2/comments')
+        .expect(200)
+        .then(({ body: { comments } }) => {
+            expect(comments).toHaveLength(0)
```

Comment:
It could also be suitable here to check the article id, to prove that its the correct article with no comments and not in fact an article which should have comments